src/commands/hardis/project/deploy/sources/dx.ts

```diff
@@ -387,13 +387,15 @@ If you need to increase the deployment waiting time (force:source:deploy --wait
       forceSourceDeployOptions,
     );
 
+    const deployExecuted = !this.checkOnly && deployXmlCount > 0 ? true : false;
+
     // Set ListViews to scope Mine if defined in .sfdx-hardis.yml
-    if (this.configInfo.listViewsToSetToMine && this.checkOnly === false) {
+    if (this.configInfo.listViewsToSetToMine && deployExecuted) {
```

Comment:
@legetz why removing the test on checkonly ? it can be equal to true with deployExecuted > 0 if we simulate a deployment during a PR check ? :)

Reply:
@nvuillam : I added new const, deployExecuted which basically means true deployment executed:
`const deployExecuted = !this.checkOnly && deployXmlCount > 0 ? true : false;`
So in the case of simulation/check, deployExecuted will never be true.

Since that already takes into account `checkOnly`, I removed that from this line. Makes sense?

Reply:
You're right, i didn't look close enough ;)